app/models/user.rb

```diff
@@ -17,6 +17,8 @@
 #  updated_at             :datetime
 #
 class User < ActiveRecord::Base
+  DEFAULT_USER_EMAIL = 'default.user@montrealrb.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.